### DISCUSSION TITLE
possibility change prompts, delete prompts and answers

### DISCUSSION THREAD

**Discussion by dmotxilla** [danny-avila/LibreChat#3840] [Feature Requests & Suggestions]
chatgpt allows the user to change (re-edit) his prompt, or detele prompts and answers. it's an awesome feature, to change the thread of the conversation.
Note: I see that it's possible to change (re-edit) a prompt in the mistral endpoint, but not in the assistants endpoint.

**Comment by danny-avila** [comment]
It's already possible here too? If not, I'm not sure what you mean

**Reply by anose001** [reply]
@danny-avila, I did not find a way to delete individual prompts, only to change them and resubmit. Is there an easy way to clear the chat instead of creating a new fork? The advantage of clearing the chat instead of creating a fork would be that I don't need to manually delete the chat by clicking on the three dots menu, pressing delete and then confirming delete. This can quickly become really time consuming as chats build over time especially when one is cognizant about long chats increasing the token count significantly, as described [here](https://community.openai.com/t/conversation-context-and-quadratic-billing/126421#:~:text=I%20think%20you,it%20works%2C%20yes.), and mentioned #3861 , #3329

**Reply by danny-avila** [reply]
There is currently no way to delete messages. The easiest way to manage context right now, on top of max context tokens as mentioned [in my comment](https://github.com/danny-avila/LibreChat/discussions/3999#discussioncomment-10617875), would be to fork the conversation, please review this article to see how much flexibility and control you have other the context: https://www.librechat.ai/docs/user_guides/fork

**Reply by anose001** [reply]
Thanks Danny!
